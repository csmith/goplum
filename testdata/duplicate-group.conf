group "test-group" {
  alert_limit = 5
  alert_window = 10m
}

group "test-group" {
  alert_limit = 3
  alert_window = 5m
}