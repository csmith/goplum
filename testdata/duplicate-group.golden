"group defined multiple times: test-group"